dicom-archive/profileTemplate.pl

```diff
@@ -33,7 +33,7 @@
 # extracts the subject and timepoint identifiers from the patient name 
 # assumes identifers are stored as <PSCID>_<DCCID>_<visit> in PatientName field, where <visit> is 3 digits.
 sub getSubjectIDs {
-    my ($patientName, $patientID, $scannerID, $dbhr, $db) = @_;
+    my ($patientName, $scannerID, $dbhr, $db) = @_;
```

Comment:
Wouldn't removing the `patientID` field in the `prod` file function `getSubjectIDs` break some projects that might be using DICOM header PatientID instead of PatientName to determine subject IDs? Unless the value `$patientName` is either `PatientName` header or `PatientID` header based on the `lookupCenterName` config value?

@nicolasbrossard thoughts on this?

Reply:
This change is not necessary if it breaks things. I just noticed it was not used in the default logic.

Reply:
OK then, if you can revert this specific change just in case there is a project that is using PatientID in their prod file. Thanks!